gensim/models/ldamodel.py

```diff
@@ -1186,9 +1224,14 @@ def load(cls, fname, *args, **kwargs):
             result.random_state = utils.get_random_state(None)  # using default value `get_random_state(None)`
             logging.warning("random_state not set so using default value")
 
+        # dtype could be absent in old models
+        if not hasattr(result, 'dtype'):
+            result.dtype = np.float64  # float64 was implicitly used before (cause it's default in numpy)
+            logging.info("dtype was not set in saved %s file %s, assuming np.float64", result.__class__.__name__, fname)
```

Comment:
Maybe `warn`?

Reply:
See https://github.com/RaRe-Technologies/gensim/pull/1656#discussion_r148215039 for discussion. Although it's expected state when loading old model, maybe a warning still a good thing.